### ISSUE TITLE
Subscribed and subscribed need to be activated / deactivated twice

### ISSUE THREAD

**Issue by rallysson** [jpuri/react-draft-wysiwyg#495]
I noticed that the envelope and the subscript need to be activated / deactivated twice to work

thanks for the great and powerful editor :)

**Comment by TheComedian08** [comment]
did u do it with One click?

**Comment by syashenkov** [comment]
Created PR to fix this bug
https://github.com/jpuri/react-draft-wysiwyg/pull/816

**Comment by TheComedian08** [comment]
No, i mean enable sub/superscript with 'OneClick' action, it still activated by 'DoubleClick'.
![draft](https://user-images.githubusercontent.com/24483169/57866490-222da900-7808-11e9-8e7d-7ed565e68a1e.gif)

**Comment by EmanuelaBolohan** [comment]
Hi @syashenkov ,
Is there any plan to fix this issue? 
Do you know when the created PR to fix this bug will be merged?

**Comment by klintSkoon** [comment]
Hi @jpuri,
I noticed this issue is persistent, is there a fix regarding it ?

**Comment by Kesavaraja** [comment]
Any update on fixing this issue ?
